src/screens/Accounts/Accounts.js

```diff
@@ -285,6 +277,11 @@ const TextContent = styled(Text)`
   padding: 0 ${spacing.medium}px 0 ${spacing.medium}px;
 `;
 
+const BannerText = styled(Text)`
+${fontStyles.medium};
+padding-left: ${spacing.small}px;
+`;
```

Comment:
Is the formatting a bit off here?

```suggestion
const BannerText = styled(Text)`
  ${fontStyles.medium};
  padding-left: ${spacing.small}px;
`;
```